scripts/reindex.py

```diff
@@ -58,6 +61,13 @@
                     ],
                     choices=config.catalogs,
                     help='The names of the catalogs to reindex.')
+parser.add_argument('--sources',
+                    default=os.environ.get('azul_reindex_sources', '*'),
```

Comment:
We should probably go through azul.Config

```suggestion
                    default=config.reindex_sources,
```